integrations/maltiverse.py

```diff
@@ -590,22 +588,73 @@ def send_event(msg: str, agent: dict = None):
         location = location.replace("|", "||").replace(":", "|:")
         event = f"1:{location}->maltiverse:{json.dumps(msg)}"
 
-    debug(event)
+    logger.debug("Event: %s", event)
+
     if len(event) > MAX_EVENT_SIZE:
-        debug(f"# WARNING: Message size exceeds the maximum allowed limit of {MAX_EVENT_SIZE} bytes.")
+        logger.warning("Message size exceeds the maximum allowed limit of %s bytes.", MAX_EVENT_SIZE)
     try:
         sock = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
         sock.connect(SOCKET_ADDR)
         sock.send(event.encode())
         sock.close()
     except socket.error as e:
         if e.errno == 111:
-            print("ERROR: Wazuh is not running.")
+            logger.error("Wazuh is not running.")
             sys.exit(6)
         elif e.errno == 90:
-            print("ERROR: Message too long to send to Wazuh.")
+            logger.error("Message too long to send to Wazuh.")
             sys.exit(7)
 
 
+def print_help_msg():
+    """Send the command's help message to the standard output."""
+    help_msg = '''
+    Exiting: Invalid arguments.
+
+    Usage:
+        maltiverse <alerts_file> <api_key> <webhook_url> [logging_level] [options_file]
+    Arguments:
+        alerts_file (required)
+            Path to the JSON file containing the alerts.
+        api_key (required)
+            Maltiverse API key.
+        webhook_url (required)
+            Maltiverse webhook URL where the messages will be sent to.
+        logging_level (optional)
+            Used to define how much information should be logged. Default is INFO.
+            Levels: NOTSET, DEBUG, INFO, WARNING, ERROR, CRITICAL.
+        options_file (optional)
+            Path to a file containing custom variables to be used in the integration. It must be JSON-encoded.
+    '''
+    print(help_msg)
+
+
+def setup_logger(args):
+    """Configure the logger.
+
+    Parameters
+    ----------
+    args: any
+        Command arguments.
+    """
+    # Create log file directories if they do not exist
+    log_file_dir = os.path.dirname(LOG_FILE)
+    if not os.path.exists(log_file_dir):
+        os.makedirs(log_file_dir)
+
+    consoleHandler = logging.StreamHandler()
+    fileHandler = TimedRotatingFileHandler(LOG_FILE, when='midnight', backupCount=31)
```

Comment:
Set a `backupCount` of 31 based on the default value used for the other logs: https://github.com/wazuh/wazuh/blob/2960662dcf23c88ea73eefdd2c330028426957aa/etc/internal_options.conf#L280-L281